LayoutTests/imported/w3c/web-platform-tests/css/css-align/parsing/align-tracks-computed.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>CSS Grid Level 3: getComputedStyle().alignContent</title>
+<link rel="help" href="https://drafts.csswg.org/css-grid-3/#tracks-alignment">
+<meta name="assert" content="align-tracks computed value is as specified.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/computed-testcommon.js"></script>
+</head>
+<body>
+
+<div id="target"></div>
+<script>
+test_computed_value("align-tracks", "normal");
+
+test_computed_value("align-tracks", "baseline");
+test_computed_value("align-tracks", "last baseline");
+
+test_computed_value("align-tracks", "space-between");
+test_computed_value("align-tracks", "space-around");
+test_computed_value("align-tracks", "space-evenly");
+test_computed_value("align-tracks", "stretch");
+
+test_computed_value("align-tracks", "center");
+test_computed_value("align-tracks", "start");
+test_computed_value("align-tracks", "end");
+test_computed_value("align-tracks", "flex-start");
+test_computed_value("align-tracks", "flex-end");
+test_computed_value("align-tracks", "unsafe end");
+test_computed_value("align-tracks", "safe flex-start");
+
+test_computed_value("align-tracks", "normal, normal")
```

Comment:
The [shortest serialization rule](https://drafts.csswg.org/cssom/#serializing-css-values) says that: `normal, normal` serializes to `normal`, and since the computed value matches the specified value. This should be:

> test_computed_value("align-tracks", "normal, normal", "normal")

Similarly, `flex-end, start, start, start` will serialize to `flex-end, start` (spec says if there are less values than the number of tracks, the last one fills the rest)

You'll probably want to add some code in ComputedStyleExtractor + StyleProperties to fix that, but feel free to leave the test failing for the initial iteration and file a new bug for this.